src/logics/Logic.cc

```diff
@@ -944,6 +944,27 @@ PTRef Logic::mkEq(vec<PTRef>& args) {
     return mkFun(eq_sym, args);
 }
 
+PTRef Logic::mkDistinct(vec<PTRef>& args) {
+    if (args.size() == 0) return getTerm_false(); // Is this correct?
+    if (args.size() == 1) return getTerm_true(); // Is this correct?
+    sort(args);
+    int i = 1;
+    int j = 0;
+    for (; i < args.size(); i++, j++) {
```

Comment:
i,j are local to the loop, can you move them inside?
It seems writing 
for (int i = 1, j = 0; i < args.size(); i++, j++) is perfectly valid.

Reply:
Hehe I never remember how the syntax goes...  You're right.